Article/Article.js

```diff
@@ -3,18 +3,18 @@
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = domElement.querySelector('.expandButton');
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.textContent = 'expand';
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.expandButton.addEventListener('click', () => this.expandArticle());
   }
 
   expandArticle() {
     // Using our reference to the domElement, toggle a class to expand or hide the article.
-
+    this.domElement.classList.toggle('article-open');
   }
```

Comment:
If you wanted to change the text of the expand button to say "close" when a tab is opened, you could drop a nice little ternary operator in there like this. If you haven't encountered ternary operators before, it's worth a Google. 

```suggestion
	 this.expandButton.innerText === 'expand' ?
      this.expandButton.innerText = 'close' :
      this.expandButton.innerText = 'expand';
  }
```